src/main/java/me/basiqueevangelist/pingspam/commands/AliasCommand.java

```diff
@@ -41,23 +41,23 @@ public static void register(CommandDispatcher<ServerCommandSource> dispatcher) {
                     .then(literal("list")
                         .executes(AliasCommand::listAliases))
                     .then(literal("add")
-                        .requires(x -> Permissions.check(x, "pingspam.addownalias", 0))
+                        .requires(x -> Permissions.check(x, "pingspam.alias.own.add", true))
```

Comment:
The permission in the README is `pingspam.alias.addown`, but here it's `pingspam.alias.own.add`.